velox/serializers/PrestoSerializer.cpp

```diff
@@ -853,6 +914,10 @@ class VectorStream {
   int32_t nullCount_{0};
   int32_t totalLength_{0};
   bool hasLengths_{false};
+  /// Indicates whether to serialize timestamps with nanosecond precision.
+  /// If false, they are serialized with millisecond precision which is
+  /// compatible with presto.
+  bool useLosslessTimestamp_{false};
```

Comment:
Make this a const and move to it ahead of non-const members along with type_? Thanks!

Reply:
done